docs/html/topics/caching.md

```diff
@@ -27,6 +27,12 @@ While this cache attempts to minimize network activity, it does not prevent
 network access altogether. If you want a local install solution that
 circumvents accessing PyPI, see {ref}`Installing from local packages`.
 
+In versions prior to 23.2, this cache was stored in a directory called `http` in
+the main cache directory (see below for its location). In 23.2 and later, a new
+cache format is used, stored in a directory called `http-v2`. If you have
+completely switched to newer versions of `pip`, you may wish to delete the old
+directory.
```

Comment:
It would be a good idea to wrap this in a `versionchanged` instead.

Reply:
Ah, did not know this existed, will do.